src/Mutator/Operator/Break_.php

```diff
@@ -77,8 +77,8 @@ public function canMutate(Node $node): bool
             return false;
         }
 
-        $parentNode = $node->getAttribute(ParentConnectorVisitor::PARENT_KEY);
+        $parentNode = ParentConnector::findParent($node);
 
-        return !$parentNode instanceof Node\Stmt\Case_;
+        return !($parentNode instanceof Node\Stmt\Case_);
```

Comment:
parentheses?

Reply:
I agree it looks strange

Reply:
what is strange to me to is to have `!` applying on the expression for `instanceof` instead of the variable like everywhere else :)

Anyway I don't want to debate about this CS: we'll stick to the convention used I'll just update it in a separate PR everywhere rather than just here

Reply:
That's the point, since we didn't agree with any of the format, we should continue using the *current* one, to avoid inconsistencies. But let's forget about this case and don't waste everyone's time...

Reply:
sorry I didn't realise this was on another thread. As said [there](https://github.com/infection/infection/pull/1224#discussion_r400072709), I'll fix it

Reply:
👍